torch/utils/cpp_extension.py

```diff
@@ -23,6 +23,13 @@
 
 
 IS_WINDOWS = sys.platform == 'win32'
+LIB_EXT = 'pyd' if IS_WINDOWS else 'so'
+EXEC_EXT = 'exe' if IS_WINDOWS else 'o'
```

Comment:
exec extension is o on linux? Really?

Reply:
I wound up just omitting it altogether. (Which I guess is the most UNIX-y thing to do.)